src/FormBuilder/Controllers/FormBuilderResourceController.php

```diff
@@ -77,6 +79,9 @@ public function update(WP_REST_Request $request)
             return rest_ensure_response($requiredFieldsError);
         }
 
+        give(UpdateEmailSettingsMeta::class)->__invoke($form);
+        give(UpdateEmailTemplateMeta::class)->__invoke($form);
+
         $form->status = $updatedSettings->formStatus;
         $form->save();
 
```

Comment:
Another note, I think we should just add an action here, `do_action('givewp_form_builder_updated', $form)` that we can add these legacy actions to so they're not hard-coded here.

Reply:
What would be the difference between adding these here versus elsewhere? There isn't another location in the code base that comes to mind without creating a new section just to replicate the above.

Regardless, I can see that the `do_action` would beneficial for extensibility in add-ons.